src/showcase/app/shared/component-viewer/stack-blitz/stack-blitz-writer.ts

```diff
@@ -82,6 +82,16 @@ export class StackBlitzWriter {
           isSbbLean ? ' class="sbb-lean"' : '',
         );
 
+        if (
+          files['src/app/journey-maps-basic-example.html'] ||
```

Comment:
I would recommend to reduce this to something like `Object.keys(files).some(f => f.includes('journey-maps')`